activesupport/lib/active_support/json/coder.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module ActiveSupport
+  module JSON
+    class Coder
+      def initialize(options = nil)
+        @options = options || {}
+      end
```

Comment:
Depending on the response to https://github.com/rails/rails/pull/54172#discussion_r1913903792, could this use keyword arguments?

```suggestion
      def initialize(**options)
        @options = options
      end
```